src/components/Leaderboard.js

```diff
@@ -2,9 +2,9 @@
 
 import * as React from 'react';
 
-type Props = {
+type Props = {|
 	leaderNames: Array<string>;

```

Comment:
This doesn't really have anything to do with this PR, but the leaderboard should probably not be passed the leaderboard names, it should run an API call to get them and store them in the state instead.